master/internal/rm/kubernetesrm/pods.go

```diff
@@ -102,9 +102,13 @@ type pods struct {
 	podInterfaces       map[string]typedV1.PodInterface
 	configMapInterfaces map[string]typedV1.ConfigMapInterface
 
+	// TODO(RM-236) make one cache and make this code more straightforward.
 	summarizeCacheLock sync.RWMutex
 	summarizeCache     summarizeResult
 	summarizeCacheTime time.Time
+	getAgentsCacheLock sync.Mutex
+	getAgentsCache     *apiv1.GetAgentsResponse
```

Comment:
This is just an in-memory cache right - aka a data structure saved in memory as opposed to an external cache, right?

Reply:
it is just in memory